src/Apps/Order/Routes/__tests__/Accept.test.tsx

```diff
@@ -54,213 +42,114 @@ const testOrder = {
   offers: { edges: Offers },
   buyer: Buyer,
 }
-let mockPushRoute: jest.Mock<string>
 
 describe("Accept seller offer", () => {
-  const getWrapper = (extraOrderProps?) => {
-    const props = {
-      relay: { environment: {} },
-      router: { push: mockPushRoute },
-      order: {
-        ...testOrder,
-        ...extraOrderProps,
-      },
-    }
-    return mount(
-      <MockBoot>
-        <AcceptRoute {...props as any} />
-        <ConnectedModalDialog />
-      </MockBoot>
-    )
-  }
-  beforeEach(() => {
-    mockPushRoute = jest.fn()
-  })
-  it("Shows the stepper", () => {
-    const component = getWrapper()
-    const stepper = component.find(OrderStepper)
-    expect(stepper.text()).toMatchInlineSnapshot(`"Respond Review"`)
-    const index = component.find(Stepper).props().currentStepIndex
-    expect(index).toBe(1)
-  })
-  it("shows the countdown timer", () => {
-    const component = getWrapper({
-      stateExpiresAt: moment(NOW)
-        .add(1, "day")
-        .add(4, "hours")
-        .add(22, "minutes")
-        .add(59, "seconds"),
-    })
-    const timer = component.find(CountdownTimer)
-    expect(timer.text()).toContain("01d 04h 22m 59s left")
-  })
-  it("shows the transaction summary", () => {
-    const component = getWrapper()
-    const transactionSummary = component.find(
-      TransactionDetailsSummaryItemFragmentContainer
-    )
-    expect(transactionSummary).toHaveLength(1)
-
-    expect(transactionSummary.text()).toMatch("Accept seller's offerChange")
-    expect(transactionSummary.text()).toMatch("Seller's offer$sellers.offer")
-  })
-  it("shows the artwork summary", () => {
-    const component = getWrapper()
-    const artworkSummary = component.find(ArtworkSummaryItemFragmentContainer)
-    expect(artworkSummary).toHaveLength(1)
-    expect(artworkSummary.text()).toMatch("Lisa BreslowGramercy Park South")
-  })
-  it("shows the shipping details", () => {
-    const component = getWrapper()
-    const shippingSummary = component.find(ShippingSummaryItemFragmentContainer)
-    expect(shippingSummary).toHaveLength(1)
-    expect(shippingSummary.text()).toMatch("Ship toJoelle Van Dyne401 Broadway")
-  })
-  it("shows the payment details", () => {
-    const component = getWrapper()
-    const paymentSummary = component.find(
-      CreditCardSummaryItemFragmentContainer
-    )
-    expect(paymentSummary).toHaveLength(1)
-    expect(paymentSummary.text()).toMatchInlineSnapshot(`"•••• 4444  Exp 3/21"`)
-  })
-  it("shows the submit button", () => {
-    const component = getWrapper()
-    const continueButton = component.find(Button).last()
-    expect(continueButton.text()).toBe("Submit")
-  })
-  it("Shows the conditions of sale disclaimer.", () => {
-    const component = getWrapper()
-    expect(component.text()).toMatch(
-      "By clicking Submit, I agree to Artsy’s Conditions of Sale."
-    )
+  const { mutations, buildPage, routes } = createTestEnv({
```

Comment:
not sure about this pattern. Would it be better for discoverability to do `const env = ...` and then use `env.mutations` etc everywhere?